src/test/erasure-code/ceph_erasure_code_non_regression.cc

```diff
@@ -70,8 +68,6 @@ int ErasureCodeNonRegression::setup(int argc, char** argv) {
      "prefix all paths with base")
     ("parameter,P", po::value<vector<string> >(),
      "add a parameter to the erasure code profile")
-    ("path", po::value<string>(), "content path instead of inferring it from parameters")
```

Comment:
the   if (vm.count("path")) below and all references to that option should also be removed